src/_zkapauthorizer/tests/fixtures.py

```diff
@@ -49,9 +47,9 @@ class AnonymousStorageServer(Fixture):
     clock = attr.ib()
 
     def _setUp(self):
-        self.tempdir = FilePath(self.useFixture(TempDir()).join(b"storage"))
+        self.tempdir = FilePath(self.useFixture(TempDir()).join(u"storage"))
         self.storage_server = StorageServer(
-            self.tempdir.asBytesMode().path,
+            self.tempdir.asTextMode().path,
```

Comment:
I'm not sure.  And I've let my local Windows dev environment fall into disrepair, I think.

On the other hand, it doesn't hurt much to be explicit about the type here right?  I know what I'm getting with `self.tempdir.asTextMode().path` but I'm not so sure if this were just `self.tempdir`.  It would depend on `FilePath` and `TempDir` and maybe also the `u"storage"` being passed in.

On Linux, looks like `TempDir.join` only accepts unicode (raises an exception when passed bytes) and returns unicode so I suppose the `asTextMode()` is superfluous here since the FilePath should already be wrapping a unicode path.

I think `StorageServer` will quickly blow up if we did manage to pass bytes here, since it calls `os.path.join` with the supplied value and a unicode string... So I guess removing the `asTextMode()` here is fine.

Reply:
It looks like there is one remaining `asBytesMode()` call in `src/_zkapauthorizer/model.py`, which is passed to `sqlite3.connect`. I looked at the code in cpython for that (in both 3.9 and 3.10), and it supports bytes, strings and anything [path-like](https://docs.python.org/3/library/os.html#os.PathLike). I'd be inclined to remove that as well.